documentation/staging/content/faq/external-clients.md

```diff
@@ -261,6 +374,7 @@ To enable an 'unknown host' source WebLogic Server to initiate EJB, JMS, or JTA
 
 #### Optional reading
 
-- For sample JMS client code and JMS configuration, see [Run Standalone WebLogic JMS Clients on Kubernetes](https://blogs.oracle.com/weblogicserver/run-standalone-weblogic-jms-clients-on-kubernetes).
+- For a discussion of the WebLogic URL syntax for JMS, EJB, and JNDI clients see [Understanding WebLogic URLs](https://docs.oracle.com/en/middleware/fusion-middleware/weblogic-server/12.2.1.4/jmsad/best_practice.html#GUID-CFA6D7B9-6059-4098-801C-30F0EE31C1AA).
```

Comment:
discussion  -> description (the referenced texts are descriptions not discussions; true for the following bulleted item as well)
For a detailed discussion -> For a detailed description

Reply:
fixed